### ISSUE TITLE
Filter column

### ISSUE THREAD

**Issue by alexchitoraga** [garygreen/pretty-routes#24]
Please add some inputs to filter methods, paths, names etc. 
Also it will be pleasant to add group filtering or maybe filtering by maps (api map, admin map, web map etc.)
_Sorry for my english_

**Comment by Braunson** [comment]
Agreed it would be nice to be able to filter down the columns. I think the PR for datatables takes care of this #10

**Comment by smujaddid** [comment]
Also sorting by route name would be nice. Datatables indeed will be nice.

**Comment by andrey-helldar** [comment]
This will be implemented in my fork of this package: https://github.com/andrey-helldar/pretty-routes/issues/5

**Comment by andrey-helldar** [comment]
Solved in my forket package: https://github.com/andrey-helldar/pretty-routes/releases
